src/json_serialization.hpp

```diff
@@ -32,8 +33,11 @@ namespace gpudrive
             }
         }
         obj.numPositions = i;
-        j.at("width").get_to(obj.width);
-        j.at("length").get_to(obj.length);
+        j.at("width").get_to(obj.vehicle_size.width);
+        j.at("length").get_to(obj.vehicle_size.length);
+        j.at("height").get_to(obj.vehicle_size.height);
+        j.at("id").get_to(obj.id); //TODO: Remove id from object struct
```

Comment:
Why do we want to remove id from object struct?

Reply:
Right now the id is in metadata and the obj struct, we need to figure out which is more appropriate. Working with metadata is way more convenient with the id in it